05 React State and working with Events/01-starting-setup/src/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+import './index.css';
+import App from './App';
+
+import {IntlProvider} from 'react-intl';
+import English from './lang/en.json';
+
+const locale = navigator.language;
+
+let lang = English;
+// if (locale==="en") {
+//    lang = English;
+// } else {
+//    if (locale === "fr") {
+//        lang = French;
+//    } else {
+//        lang = Arabic;
+//    }
+// }
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <IntlProvider locale ={locale} messages={English}>
```

Comment:
Have you understood what is the concept of react used here. 
Root <App> component its getting enclosed inside 

 <IntlProvider locale ={locale} messages={English}>
 
How do you define this in React?